packages/in-app-notification-dispatcher/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "pagopa-interop-in-app-notification-dispatcher",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Pagopa Interop In-App Notification Dispatcher",
+  "main": "dist",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "test": "vitest --config vitest.config.ts",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "tsx -r 'dotenv-flow/config' --watch  ./src/index.ts",
+    "build": "tsc && pnpm cpx './src/resources/**/*' './dist/resources'",
```

```suggestion
    "build": "tsc",
```
there is no `resources` dir, right?